src/validation/validation.cpp

```diff
@@ -3186,8 +3208,6 @@ bool ActivateBestChainStep(CValidationState &state,
                 }
             }
         }
-        if (fInvalidFound)
-            break; // stop processing more blocks if the last one was invalid.
```

Comment:
why remove?  I think at the very least, the next line (notify the UI) should not be run if an invalid block is found (since there's no new tip)... subsequent lines also seem unnecesssary.  Is there a reason to continue processing blocks beyond an invalid one?

Reply:
fixed